.gitignore

```diff
@@ -15,3 +15,4 @@ Gemfile.lock
 .covered.db
 .DS_Store
 /gemfiles
+.idea
```

Comment:
What’s this file format? Happy to include it here if it’s helpful to anyone. Just wondering what it is.

Reply:
That's the config when using RubyMine